addons/sky_3d/src/Skydome.gd

```diff
@@ -471,62 +388,27 @@ func update_moon_coords() -> void:
 ## Atmosphere
 #####################
 
-var atm_quality: int = 1: set = set_atm_quality
-var atm_wavelenghts:= Vector3(680.0, 550.0, 440.0): set = set_atm_wavelenghts
-var atm_darkness: float = 0.5: set = set_atm_darkness
-var atm_sun_intensity: float = 18.0: set = set_atm_sun_intensity
-var atm_day_tint:= Color(0.807843, 0.909804, 1.0): set = set_atm_day_tint
-var atm_horizon_light_tint:= Color(0.980392, 0.635294, 0.462745, 1.0): set = set_atm_horizon_light_tint
-var atm_enable_moon_scatter_mode: bool = false: set = set_atm_enable_moon_scatter_mode
-var atm_night_tint:= Color(0.168627, 0.2, 0.25098, 1.0): set = set_atm_night_tint
-var atm_level_params:= Vector3(1.0, 0.0, 0.0): set = set_atm_level_params
-var atm_thickness: float = 0.7: set = set_atm_thickness
-var atm_mie: float = 0.07: set = set_atm_mie
-var atm_turbidity: float = 0.001: set = set_atm_turbidity
-var atm_sun_mie_tint:= Color(1.0, 1.0, 1.0, 1.0): set = set_atm_sun_mie_tint
-var atm_sun_mie_intensity: float = 1.0: set = set_atm_sun_mie_intensity
-var atm_sun_mie_anisotropy: float = 0.8: set = set_atm_sun_mie_anisotropy
-var atm_moon_mie_tint:= Color(0.137255, 0.184314, 0.292196): set = set_atm_moon_mie_tint
-var atm_moon_mie_intensity: float = 0.7: set = set_atm_moon_mie_intensity
-var atm_moon_mie_anisotropy: float = 0.8: set = set_atm_moon_mie_anisotropy
-
-
-func set_atm_quality(value: int) -> void:
-	if value == atm_quality:
-		return
-	atm_quality = value
-	update_atm_quality()
-
-
 func update_atm_quality() -> void:
 	if !is_scene_built:
 		return
-	if atm_quality == SkyQuality.Low:
-		sky_material.shader = Sky3D._sky_shader
-		sky_sphere.radial_segments = 16
-		sky_sphere.rings = 8
-	else:
-		sky_material.shader = Sky3D._pv_sky_shader
-		sky_sphere.radial_segments = 64
-		sky_sphere.rings = 64
+	sky_material.shader = Sky3D._new_sky_shader
 
 
-func set_atm_wavelenghts(value : Vector3) -> void:
-	if value == atm_wavelenghts:
+func set_atm_wavelengths(value : Vector3) -> void:
+	if value == atm_wavelengths:
 		return
-	atm_wavelenghts = value
+	atm_wavelengths = value
 	update_beta_ray()
 	
 
 func update_beta_ray() -> void:
 	if !is_scene_built:
 		return
 
-	var wll = ScatterLib.compute_wavelenghts_lambda(atm_wavelenghts)
+	var wll = ScatterLib.compute_wavelenghts_lambda(atm_wavelengths)
 	var wls = ScatterLib.compute_wavlenghts(wll)
 	var betaRay = ScatterLib.compute_beta_ray(wls)
```

Comment:
Please add types for all of these. The styling you've been doing everywhere is great.